src/containers/UserSettings/UserSettings.tsx

```diff
@@ -30,13 +24,16 @@ export const UserSettings = ({settings: userSettings = settings}: UserSettingsPr
                                     {sectionSettings.map((setting) => {
                                         if (setting.type === 'info') {
                                             return (
-                                                <SettingsInfoField
-                                                    key={setting.title}
-                                                    {...setting}
-                                                />
+                                                <Settings.Item key={setting.title} {...setting}>
+                                                    {setting.content}
+                                                </Settings.Item>
                                             );
                                         }
-                                        return <Setting key={setting.settingKey} {...setting} />;
+                                        return (
+                                            <Settings.Item key={setting.title} {...setting}>
+                                                <Setting {...setting} />
+                                            </Settings.Item>
+                                        );
```

Comment:
Previously, there was I problem, that if `Settings.Item` was not a direct child of `Settings.Section`, `title` was not rendered properly. To fix this, I mistakenly used `highlightedTitle`, so search didn't highlight anything